### ISSUE TITLE
'Open in terminal' not focusing window

### ISSUE THREAD

**Issue by TheGreatLambino** [fcaviggia/classification-banner#20]
Prior to installation on a CentOS 7.2.1511 VM, I was able to right-click, select 'Open in terminal', and begin typing into the terminal window because it was focused. After installation, I have to click inside the terminal window to begin typing because the focus is not happening automatically. 

I tried modifying the classification-banner.desktop file, since I saw a reference to terminal, and I changed the terminal=false to terminal=true, but this just launched an active terminal window at login, but it did not present the classification banner in the desktop.

**Comment by fcaviggia** [comment]
What happens when you run it manually from the terminal (e.g. $ ./classification-banner) - any errors?

**Comment by TheGreatLambino** [comment]
No, I'm not getting any errors when I login or when I run it manually. The
only thing I'm noticing is that when I right-click in the GDM, and choose
'Open in Terminal', I get the message in the bottom toolbar that reads
'Terminal "admin@localhost:~/Desktop" is ready', and the terminal window
appears; however, the focus doesn't change to the new window automatically.
I have to click inside of it in order to begin typing.

Before I installed the classification-banner, I did not have to click
inside the new window in order to start typing inside of it.
